balboa-client-kafka/src/main/scala/com/socrata/metrics/producer/BalboaKafkaProducer.scala

```diff
@@ -0,0 +1,94 @@
+package com.socrata.metrics.producer
+
+import java.util.Properties
+
+import com.socrata.balboa.common.kafka.Constants
+import com.socrata.balboa.common.kafka.util.AddressAndPort
+import kafka.producer.{KeyedMessage, Producer, ProducerConfig}
+import kafka.serializer.Encoder
+
+/**
+ * A wrapper around Kafka [[Producer]].  This Producer is restricted to exactly one Kafka Topic.  The producer is
+ * currently this producer is configured to send messages Syncronously (See: [[Constants.producerDefaultProps]]).
+ *
+ * @tparam K Kafka Key type.  Can be anything as long as the Encoder is appropiately configured
+ * @tparam M Kafka Message type.  Can be anything as long as the Encoder is appropiately configured
+ */
+trait BalboaKafkaProducer[K,M] {
+
+  val producer: Producer[K,M]
```

Comment:
I just made that into a base class.  producers are now evaluated/constructed at wrapper construction time.